.github/workflows/ci.yml

```diff
@@ -74,3 +74,73 @@ jobs:
         run: |
           aws s3 cp ${PROJECT}_${NEW_VERSION}.bin s3://dev-vitro-shard-applications/bumped-versions/
           aws s3 cp ${PROJECT}_${NEW_VERSION}_upgrade_package.swu s3://dev-vitro-core-sys-images/
+
+  build-telementry:
+    name: Build telementry test app
```

Comment:
@Stojek139808 should it not be `telemetry`?

Reply:
as below, fixed